app/schemas/models/cinematic.schema.js

```diff
@@ -0,0 +1,151 @@
+const c = require('./../schemas')
+
+const CharacterSchema = (title) => c.object({
+  title: title,
+  description: 'ThangType that will appear on the right side of the screen.',
+  required: ['type']
+}, {
+  type: c.shortString({
+    title: 'Type',
+    description: 'Whether this is the player hero, a thangType slug or null.',
+    enum: ['slug', 'hero', 'null'],
+    default: 'null'
+  }),
+  slug: c.shortString({
+    title: 'ThangType Slug',
+    description: 'Required if type is set to `slug`'
+  }),
+  enterOnStart: {
+    type: 'boolean',
+    title: 'Animate in?',
+    description: 'If true the character will animate in. Otherwise the character will start simply there.'
+  },
+  position: c.point2d({ title: 'Position', description: 'Where character is located.' })
+})
+
+const ShotSetup = c.object({
+  title: 'ShotSetup'
+}, {
+  cameraType: c.shortString({
+    title: 'Camera Type',
+    description: 'The shot type',
+    enum: ['right-close', 'left-close', 'dual'],
+    default: 'dual'
+  }),
+  rightThangType: CharacterSchema('Right Character'),
+  leftThangType: CharacterSchema('Left Character'),
+  backgroundArt: c.object({
+    title: 'Background Art',
+    description: 'The art in the background of this shot.'
+  }, {
+    type: c.shortString({ enum: ['slug', 'null'] }),
+    slug: c.shortString({
+      title: 'Background path',
+      description: 'Path to the background asset'
+    })
+  })
+  // TODO: music
+  // TODO: next - id
+})
+
+const DialogNode = c.object({
+  title: 'Dialog Node',
+  description: 'A node of a shot. Contains dialog instructions.',
+  required: ['dialogClear']
+}, {
+  speaker: c.shortString({ enum: ['left', 'right'], title: 'Speaker', description: 'Which character is speaking. Used to select speech bubble.' }),
+  text: { type: 'string', title: 'Text', description: 'html text. TODO: how to translate this and interpolate this???' },
```

Comment:
Why is the TODO comment in the description?

Reply:
I will remove comment and add it here as a comment. Thanks